hat/assets/js/apps/Iaso/components/files/pdf/DocumentUploadWithPreview.tsx

```diff
@@ -34,14 +36,28 @@ const DocumentUploadWithPreview: React.FC<DocumentUploadWithPreviewProps> = ({
     return (
         <Grid container spacing={2} alignItems="center">
             <Grid item xs={document ? 10 : 12}>
-                <FilesUpload
-                    accept={acceptPDF}
-                    files={document ? [document as unknown as File] : []}
-                    onFilesSelect={onFilesSelect}
-                    multi={false}
-                    errors={errors}
-                    placeholder={formatMessage(MESSAGES.document)}
-                />
+                {disabled ? (
+                    <input
+                        type="text"
+                        value={
+                            pdfUrl
+                                ? 'Document attached'
+                                : 'No document attached'
```

Comment:
This could be translated.

Also I don't know how much effort it takes, but as suggested above it would be better to handle that at the `FilesUpload` component level.